src/app/tests/suites/TestBasicInformation.yaml

```diff
@@ -171,17 +171,39 @@ tests:
     - label: "Read the Specification Version value"
       command: "readAttribute"
       attribute: "SpecificationVersion"
+      minRevision: 3
       response:
           # For now all-clusters-app has a version 1.5
           value: 0x01050000
 
     - label: "Read the Max Paths Per Invoke value"
       command: "readAttribute"
       attribute: "MaxPathsPerInvoke"
+      minRevision: 3
       response:
           constraints:
               minValue: 1
 
+    - label:
+          "Force-fail if minRevision is not handled properly (should be skipped
+          by minRevision). Used to test YAML minRevision support."
+      command: "readAttribute"
+      attribute: "MaxPathsPerInvoke"
+      minRevision: 100
+      response:
+          constraints:
+              maxValue: 0
+
+    - label:
+          "Force-fail if maxRevision is not handled properly (should be skipped
+          by maxRevision). Used to test YAML maxRevision support."
+      command: "readAttribute"
+      attribute: "MaxPathsPerInvoke"
+      maxRevision: 1
```

Comment:
should we use 0? Or did we never release a basicinfo at revision 1. Maybe some explanations in the label on why 1 and 100 are considered "never true" would be helpful (and really hoping we do not reach revision 100 to need to update the above test...)

Reply:
Fixed